apps/i18n/common/en_us.json

```diff
@@ -2869,6 +2869,13 @@
   "progressV2_feedback_shareMoreLinkText": "Share more feedback",
   "progressV2_feedback_thumbsUp": "I like it!",
   "progressV2_feedback_thumbsDown": "I don't like it",
+  "submitProjectGallery_alreadySubmitted": "Submitted to Featured Projects gallery",
+  "submitProjectGallery_details": "*Submitting a project does not guarantee that it will be featured. Once submitted, this project cannot be submitted again.",
+  "submitProjectGallery_exampleText": "Example text",
+  "submitProjectGallery_goBack": "Go back",
+  "submitProjectGallery_header": "Want to show the world what you made?",
+  "submitProjectGallery_submit": "Submit to be featured",
+  "submitProjectGallery_tellUs": "Tell us about your project!",
```

Comment:
If this is referring to the text box after it, how about ending with a `:`?

Reply:
@samantha-code - lmk what you think.

Reply:
We can do "Tell us about your project: "